src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Object> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("set의 크기를 확인한다.")
+    @Test
+    void set_size() {
+        int expected = 3;
+        assertEquals(expected, numbers.size());
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
JUnit이 아닌 assetj의 assertThat 활용할 것을 추천

Reply:
JUnit이 아닌 assetj의 assertThat 활용하는 것이 더 좋은 이유가 있을까요?
assertEqauls나 assertTrue 등이 짧아서 자주 사용하곤 했는데 혹시 가독성 등의 측면에서 안좋은 점이 있을까해서요!